Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
@emartinez167   I fully understand your statements.  We would like you to give us a second chance.   If you even think you were treated less politely than you should have been, please send an @ message to ThinkyHead or me.   My guess is we won't be getting any messages from you going forward.  We are very serious about keeping this a healthy environment for both users and developers.